src/System.Collections.NonGeneric/tests/Stack/StackTests.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
thanks for catching this.  these two files shouldn't actually be here.  let me --amend history :angel: